### PR TITLE
feat: Creation of mapping between all CRD entitites

### PR DESCRIPTION
**Pull request by SujeethKumarAri**
### JIRA link  ###
https://tools.hmcts.net/jira/browse/RDCC-2014

### Change description ###
Creation of mapping between all CRD entitites. 
Please refer https://tools.hmcts.net/confluence/display/RTRD/Case+Worker+Reference+Data+-++Data+Model?preview=/1457301758/1460553121/image2020-11-2_22-9-34.png for the Data table

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+
+@Entity(name = "case_worker_location")
+@Getter
+@Setter
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@SequenceGenerator(name = "case_worker_location_id_seq",
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java:32]: I think sequence not needed in many tables apart from main profile table. As it serial and postgress take care of it.

**Review by abhijitdiwan — COMMENTED** [review]
Please follow git commit conventions.

File: src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java (hunk 1 above)
Comment (SujeethKumarAri) [reply src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerLocation.java:32]: Sure will remove sequence. thanks

**Comment by SujeethKumarAri** [comment]
Closing this PR as I messed up while rebasing. I have created a new PR https://github.com/hmcts/rd-caseworker-ref-api/pull/70. Apologies

**Pull request closed without merging**